### ISSUE TITLE
Default values for Array and Struct

### ISSUE THREAD

**Issue by benedeki** [AbsaOSS/enceladus#1230]
## Background
When a non-null default (fallback) value for complex types (array, struct) would be requested during the **Standardization** it will stop the process with exception because the value is not defined.

## Feature
And default (fallback) value for Array and Struct types

## Proposed Solution [Optional]
Array: empty array
Struct: each field is fllled with default value of the fields type

**Comment by benedeki** [comment]
Make `TypeParser.defaultValue` a `val`

**Comment by yruslan** [comment]
👍 An empty array is a good default fallback value for arrays.
But is the current behavior for structs the same as described?

**Comment by benedeki** [comment]
No, the behavior would be changed. But from crash to actually finishing the job, with error in _errCol_, so IMO legit change, without actual effect on current deploys.
